draft-ietf-httpbis-http2bis.xml

```diff
@@ -4288,18 +4174,19 @@
             </dd>
```

Comment:
Up there^^ probably best to say that this updates it to obsolete it.  I think that you could just say something like "this marks the header field as as obsoleted" and avoid the registration block.  A backward reference to its original definition is probably all you need in addition to that.